Changing the postfix-files config file won't work right on upgrades because
it's a noreplace configuration (the new version with the right paths in it is
installed as postfix-files.rpmnew), so we just munge the post-install script.
--- postfix-1.1.3/conf/post-install	Thu Feb 28 16:15:08 2002
+++ postfix-1.1.3/conf/post-install	Thu Feb 28 16:14:54 2002
@@ -429,6 +429,16 @@
 	case $path in
 	no|no/*) continue;;
 	esac
+	# Munge paths for alternatives.
+	case $path in
+		/usr/bin/mailq) path=$path.postfix ;;
+		/usr/bin/newaliases) path=$path.postfix ;;
+		/usr/bin/rmail) path=$path.postfix ;;
+		/usr/sbin/sendmail) path=$path.postfix ;;
+		/usr/share/man/man1/mailq.1.gz) path=/usr/share/man/man1/mailq.postfix.1.gz ;;
+		/usr/share/man/man1/newaliases.1.gz) path=/usr/share/man/man1/newaliases.postfix.1.gz ;;
+		/usr/share/man/man5/aliases.5.gz) path=/usr/share/man/man5/aliases.postfix.5.gz ;;
+	esac
 	# Pick up the flags.
 	case $flags in *u*) upgrade_flag=1;; *) upgrade_flag=;; esac
 	case $flags in *c*) create_flag=1;; *) create_flag=;; esac
